dev-packages/e2e-tests/test-applications/angular-17/src/main.ts

```diff
@@ -9,7 +9,7 @@ Sentry.init({
   tracesSampleRate: 1.0,
   integrations: [Sentry.browserTracingIntegration({})],
   tunnel: `http://localhost:3031/`, // proxy server
-  debug: true,
+  debug: !!process.env.DEBUG,
```

Comment:
Made this opt-in everywhere to not pollute logs in regular runs.

Reply:
this might break - I think angular isn't smart enough to reason about `process.env` at build time 😬 
But we'll see it anyway in CI 😅

Reply:
IIRC I hard-coded the DSN a couple of lines above for the same reason (should have left a comment)

Reply:
ah, I see xD well if it break I will remove it :D

Reply:
it failed locally as well now, so I removed this - and also added a comment for future us :D